.github/workflows/ci-test-go.yml

```diff
@@ -109,6 +109,16 @@ jobs:
         working-directory: ./${{ inputs.project-directory }}
         run: go build
 
+      - name: Install dependencies
+        shell: bash
+        run: |
+          SCRIPT_PATH="./.github/scripts/${{ inputs.project-directory }}/install-dependencies.sh"
```

Comment:
Because not all modules need to contribute how to start dependencies. This shell script offers an extension point to add something to the CI, if needed, that's why it uses the input to generate the path, and if it does not exist (in this PR for all the modules but Ollama), do nothing